.env.sample

```diff
@@ -0,0 +1,4 @@
+# GitHub credentials to access the repository repository where you are going to track the deployments.
```

Comment:
I would suggest to use this gem https://github.com/bkeepers/dotenv to load the env variables, then we don't have to load the file manually

Reply:
After some conversations offline, we decided not to use `dotenv` to simplify things when using this gem inside the `obs-ansible` project.

Reply:
Could you explain how using _dotenv_ complicates using _obs_github_deployments_ in _obs-ansible_? The alternative to _dotenv_ is manually loading the environment variables and it isn't great.